Includes/menu.hpp

```diff
@@ -39,6 +41,12 @@ class MenuNode : public MenuItem {
 
   void up();
   void down();
+  void pageup();
+  void pagedown();
```

Comment:
Please consider using lowerCamelCase for methods.

Reply:
Done.